changelog.md

```diff
@@ -69,6 +69,15 @@
 
 - nil dereference is not allowed at compile time. `cast[ptr int](nil)[]` is rejected at compile time.
 
+- An enum now can't be converted to another enum directly, you must use `ord` (or `cast`, but
+  compiler won't help if you misuse it).
+  ```
+  type A = enum a1, a2
+  type B = enum b1, b2
+  doAssert not compiles(a1.B)
+  doAssert compiles(a1.ord.B)
+  ```
+  for a transition period, use `-d:nimLegacyConvEnumEnum`.
```

Comment:
I think this should be put under a "Breaking Changes" heading

Reply:
there used to be one in changelog template but it got removed

Reply:
done, added: `Changes affecting backward compatibility`